spec/rmagick/draw_spec.rb

```diff
@@ -0,0 +1,157 @@
+class Magick::Draw
+  def self._dummy_img_
+    @@_dummy_img_
+  end
+end
+
+describe Magick::Draw do
+  before(:each) { @draw = Magick::Draw.new }
```

Comment:
Turns out `let` worked without the `!`. I normally use `let`, but I can see the benefit of instance variables since they make the so-called "mystery guests" a little more apparent. We also use the [rom style](https://github.com/solnic/rom-mapper/tree/master/spec/unit/rom/mapper) spec structure, one spec file per method, which also reduces the overhead, since the `let`s only apply to the current method.